regression-test/suites/nereids_rules_p0/grouping_sets/grouping_normalize_test.groovy

```diff
@@ -39,4 +39,19 @@ suite("grouping_normalize_test"){
     SELECT  ROUND( SUM(pk  +  1)  -  3)  col_alias1,  MAX( DISTINCT  col_int_undef_signed  -  5)   AS col_alias2, pk  +  1  AS col_alias3
     FROM grouping_normalize_test  GROUP BY  GROUPING SETS ((col_int_undef_signed,col_int_undef_signed2,pk),()) order by 1,2,3;
     """
+
+    explain {
```

Comment:
also add a plan check test in [NormalizeRepeatTest.java](https://github.com/apache/doris/pull/35813/files#diff-96ad3ba35e43204328613a2967f79313c2caaf1e76d8bacd1a322768c542c578)